.github/workflows/prerelease.yml

```diff
@@ -153,3 +165,40 @@ jobs:
           slack-bot-user-oauth-access-token: ${{ secrets.COREINT_SLACK_TOKEN }}
           slack-channel: ${{ secrets.COREINT_SLACK_CHANNEL }}
           slack-text: "❌ `${{ env.REPO_FULL_NAME }}`: prerelease pipeline failed."
+  
+  publish-to-s3:
+    name: Send release assets to S3
+    runs-on: ubuntu-20.04
+    needs: [package-win]
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+      - name: Publish to S3 action
+        uses: newrelic/infrastructure-publish-action@v1
+        env:
+          AWS_S3_BUCKET_NAME: "nr-downloads-ohai-staging"
+          AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock-staging"
+          AWS_REGION: "us-east-1"
+          DISABLE_LOCK: false
```

Comment:
I'd vote for stripping this, both env var and input. It's the default (with no reason for it to change) and shouldn't be overridden unless for very specific tests.

Reply:
I get it. But in case you need to change it, are you going to remember the input name/env var? You'll have to look at the action to remember what was the name of the inputs.